workflow-service-sdk/README.md

```diff
@@ -0,0 +1,155 @@
+# workflow-service-sdk
+
+Parodos Workflow Service API
+- API version: v1.0.0
+  - Build date: 2023-03-17T18:03:38.027364+01:00[Europe/Rome]
+
+This is the API documentation for the Parodos Workflow Service. It provides operations to execute assessments to determine infrastructure options (tooling + environments). Also executes infrastructure task workflows to call downstream systems to stand-up an infrastructure option.
+
+
+*Automatically generated by the [OpenAPI Generator](https://openapi-generator.tech)*
+
+
+## Requirements
+
+Building the API client library requires:
+1. Java 1.7+
+2. Maven (3.8.3+)/Gradle (7.2+)
+
+## Installation
+
+To install the API client library to your local Maven repository, simply execute:
+
+```shell
+mvn clean install
+```
+
+To deploy it to a remote Maven repository instead, configure the settings of the repository and execute:
+
+```shell
+mvn clean deploy
+```
+
+Refer to the [OSSRH Guide](http://central.sonatype.org/pages/ossrh-guide.html) for more information.
+
+### Maven users
+
+Add this dependency to your project's POM:
+
+```xml
+<dependency>
+  <groupId>dev.parodos</groupId>
+  <artifactId>workflow-service-sdk</artifactId>
+  <version>v1.0.0-SNAPSHOT</version>
```

Comment:
version schema should match the existing code-base which is 1.0.2-SNAPSHOT.
Is this file auto-generated?

Reply:
Yes, It's auto generated.
Fixed!